AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/LocalStorage/AnalyticsEventSQLStorage.swift

```diff
@@ -100,6 +100,22 @@ class AnalyticsEventSQLStorage: AnalyticsEventStorage {
         let deleteStatement = "DELETE FROM Event"
         _ = try dbAdapter.executeQuery(deleteStatement, [])
     }
+    
+    func updateEvents(ofType eventType: String,
+                      withSessionId sessionId: PinpointSession.SessionId,
+                      setAttributes attributes: [String : String]) throws {
+        let updateStatement = """
+        UPDATE Event
+        SET attributes = ?
+        WHERE sessionId = ?
+        AND eventType = ?
+        """
+        _ = try dbAdapter.executeQuery(updateStatement, [
+            PinpointEvent.archiveEventAttributes(attributes),
```

Comment:
This is gonna conflict with the changes in https://github.com/aws-amplify/amplify-ios/pull/1975, as now the attributes are stored as a Blob instead of a String

Reply:
true, it'd be better to merge #1975 first and then rebase on top of that

Reply:
Rebased on dev-preview and updated to use `Blob`